### PR TITLE
Properly handle tvOS targets in scan.

### PR DESCRIPTION
**Pull request by mpirri**

### CODE REVIEW

File: lib/scan/detect_values.rb [hunk 1]
```diff
@@ -56,6 +57,40 @@ def self.default_device
       raise "No simulators found".red unless Scan.device
     end
 
+    def self.default_device_tvos
+      config = Scan.config
+
+      if config[:device] # make sure it actually exists
+        device = config[:device].to_s.strip.tr('()', '') # Remove parenthesis
+
+        found = FastlaneCore::Simulator.all(requested_os_type: 'tvOS').find { |d| (d.name + " " + d.tvos_version).include? device }
```
Comment (KrauseFx) [inline lib/scan/detect_values.rb]: You can change this to be multi-line using the `do` and `end` block to make it easier to read :+1:

**Comment by KrauseFx** [comment]
This looks great! I haven't tested it with a tvOS project, but the code looks good :+1:

**Comment by KrauseFx** [comment]
What's the status on this?

**Comment by mpirri** [comment]
Found a bug in the related scan PR, I want to get that fixed, and then I'll have you do one final look-over.

**Comment by KrauseFx** [comment]
Thanks @mpirri :+1:

**Pull request merged**